tests/autoyast/installation.pm

```diff
@@ -139,6 +139,7 @@ sub run {
         @expected_warnings = @{$test_data->{expected_warnings}};
         push(@needles, @expected_warnings);
     }
+    my @processed_warnings;
```

Comment:
If you would like you could move this in L186 to have the variables in one place and help readability. But no problem in general

Reply:
it is in the right place, after @expected_warnings, both should closed to understand the meaning.